src/vs/workbench/api/common/extHostNotebook.ts

```diff
@@ -382,6 +386,10 @@ export class ExtHostNotebookEditor extends Disposable implements vscode.Notebook
 	createCell(content: string, language: string, type: CellKind, outputs: vscode.CellOutput[], metadata: vscode.NotebookCellMetadata | undefined): vscode.NotebookCell {
 		const handle = ExtHostNotebookEditor._cellhandlePool++;
 		const uri = CellUri.generate(this.document.uri, handle);
+		metadata = {
+			...notebookCellMetadataDefaults,
```

Comment:
@roblourens This will break the default cell editable metadata set on notebook document. For example

```json
{
  metadata: {
    cellEditable: false
  },
  cells: [
     { } 
  ]
}
```

Cell will always be `editable` as we will set a default value for it.

We may want to use `undefined/null` here if metadata is not set on a cell. I'll make changes on master.

Reply:
I didn't know there is default cellEditable metadata, I don't see that used anywhere.

Reply:
oh, that was added by me later on. False alarm for this PR but just FYI.